source/v4/posts.yaml

```diff
@@ -70,6 +70,31 @@
           $ref: '#/responses/Unauthorized'
         '403':
           $ref: '#/responses/Forbidden'
+    delete:
+      tags:
+        - posts
+      summary: Deletes a post
+      description: |
+        Deletes a single post.
```

Comment:
Can you update this to be "Soft deletes a post, by marking the post as deleted in the database. Soft deleted posts will not be returned in post queries."? Later we'll add endpoints for permanently deleting posts